feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -101,6 +100,7 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
   isAsmCustomer360Configured: boolean | undefined = false;
   isAsmCustomer360Loaded$ = new BehaviorSubject<boolean>(false);
   protected featureModules = inject(FeatureModulesService);
+  protected featureToggles = inject(FeatureToggles);
```

Comment:
`major:` `FeatureToggles`  is supposed to be used only in factories that provides a `Config` based on the feature flag, to avoid circular dependency. In other places, e.g components, let's use `FeatureConfigService` instead.

Apart from that, let's change the access modifier to `private` so customers cannot build their logic on top of the interim service.

```suggestion
  private featureConfigService = inject(FeatureConfigService);
```